pkg/odo/genericclioptions/context.go

```diff
@@ -30,6 +30,26 @@ func NewContextCreatingAppIfNeeded(command *cobra.Command) *Context {
 	return newContext(command, true, false)
 }
 
+// NewConfigContext is a special kind of context which only contains local configuration, other information is not retrived
+//  from the cluster. This is useful for commands which don't want to connect to cluster.
+func NewConfigContext(command *cobra.Command) *Context {
```

Comment:
Should we name this `NewLocalConfigContext` or `NewLocalConfig` since it's specifically about local configuration?

Reply:
NewLocalConfigContext sounds better

Reply:
resolved